ext/unmixer/2.6.0/stubs.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __UNMIXER_RUBY_STUBS__
+#define __UNMIXER_RUBY_STUBS__
```

Comment:
Please change `__UNMIXER_RUBY_STUBS__` to `UNMIXER_RUBY_STUBS`.
`__UNMIXER_RUBY_STUBS__` is reserved names.

Reply:
done :)